react/features/transcribing/reducer.ts

```diff
@@ -44,6 +45,23 @@ export interface ITranscribingState {
 ReducerRegistry.register<ITranscribingState>('features/transcribing',
     (state = _getInitialState(), action): ITranscribingState => {
         switch (action.type) {
+        case CONFERENCE_PROPERTIES_CHANGED: {
+            const audioRecordingEnabled = action.properties?.['audio-recording-enabled'] === 'true';
+
+            if (!state.isTranscribing && audioRecordingEnabled) {
```

Comment:
```suggestion
            if (state.isTranscribing != audioRecordingEnabled) {
                return {
                    ...state,
                    isTranscribing: audioRecordingEnabled
                };
            }
            return state;
```

Reply:
It's intentional that you don't just always return `{ ...state, isTrainscribing: audioRecordingEnabled }`, right?